src/main/java/org/prebid/server/bidder/rubicon/proto/RubiconVideoExt.java

```diff
@@ -12,4 +14,8 @@
     Integer skipdelay;
 
     RubiconVideoExtRp rp;
+
+    @JsonProperty("videotype")
+    VideoType videoType;
+
```

Comment:
Please remove an extra blank line after last variable declaration. There should be an empty line after last symbol in each `.java` class, which in 99% cases is a closing curly bracket of the class.